### ISSUE TITLE
Feature Request: Vertex (and/or Face) Color => Material + texture

### ISSUE THREAD

**Issue by nine7nine** [cloudcompare/trunk#444]
Hey CC,

I had an idea for a feature that would be nice to have in CC.

I'd like to be able to easily transfer/convert colors from vertex and/or faces into texture/material for a mesh... Kind of similar to the menu option; edit=>colors=>'interpolate color from another entity' to some degree...

Currently, unless I am missing something (?), there isn't an easy/straightforward way to do this... I've bumped into this a few times, where I have created a point cloud from a mesh, then created a new mesh from the point cloud ~ but wanted to, either;

1. use the vertex color to create a material / texture for the mesh
2. or transfer the material/texture from the original onto the new mesh... 

This kind of thing can be done in blender3d, but blender, IMO, is a bit of a pain to use / not user friendly compared to CC and the workflow leaves a lot to be desired... I think it would be interesting if this sort of thing could be done in CC.

food for thought. Thanks.

**Comment by cloudcompare** [comment]
Indeed, you can't do this in CC. But I think Meshlab has this ability, hasn't it?

**Comment by nine7nine** [comment]
Meshlab, I think it is supposed to. however, I've never had any success in getting it to work correctly.

I can get the vertex=>face color to work in Meshlab, but not actually getting the vertex/face=> texture/material to work correctly. 

Not only that, But Meshlab is fairly 'crashy' on Linux, just a ton of bugs in v1.3.3, memory leaks, etc... I've tried getting it Meshlab/svn to compile; it requires a number of patches for linux that have been around for years, (and now a gcc6 patch too) and their build system is somewhat broken (on Linux anyway. they should switch to cmake). So while, I do have Meshlab installed - I tend to avoid using it. 

Ont top of that CC tends to be stable and there's also stuff that CC just does 'better', more efficiently and/or faster ~ like Poisson Reconstruction for example; Meshlab eats memory with Poisson, tells me to install a 64bit version with some settings (meanwhile, i am using the 64bit version and have 16gb or RAM. lol), can hang the machine (swapping to disk), under conditions where CC works just fine.

(as you can probably tell, I'm not a huge fan of Meshalb, love CC though!)
